docs/fileset-catalog.md

```diff
@@ -21,6 +21,120 @@ Note that Gravitino uses Hadoop 3 dependencies to build Fileset catalog. Theoret
 compatible with both Hadoop 2.x and 3.x, since Gravitino doesn't leverage any new features in
 Hadoop 3. If there's any compatibility issue, please create an [issue](https://github.com/apache/gravitino/issues).
 
+In general, the locations of all schemas and filesets under a fileset
+catalog belong to a single Hadoop cluster if they are HDFS location.
+
+The example for creating a fileset is as follows:
+```text
+# create fileset catalog
+curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
+-H "Content-Type: application/json" -d '{
+  "name": "fileset_catalog",
+  "type": "FILESET",
+  "comment": "This is a fileset catalog",
+  "provider": "fileset",
+  "properties": {
+    "location": "hdfs://172.17.0.2:9000/fileset_catalog"
+  }
+}' http://localhost:8090/api/metalakes/test/catalogs
+
+# create a fileset schema under the catalog with inherited properties
+curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
+-H "Content-Type: application/json" -d '{
+  "name": "test_schema",
+  "comment": "This is a schema",
+  "properties": {
+  }
+}' http://localhost:8090/api/metalakes/test/catalogs/fileset_catalog/schemas
+
+# create a fileset under the schema with inherited properties
+curl -X POST -H "Accept: application/vnd.gravitino.v1+json"
+-H "Content-Type: application/json" -d '{
+  "name": "fs1",
+  "comment": "This is an example fileset",
+  "type": "MANAGED",
+  "properties": {
+  }
+}' http://localhost:8090/api/metalakes/test/catalogs/fileset_catalog/schemas/test_schema/filesets
+```
```

Comment:
Is this part duplicated with that one?
https://github.com/apache/gravitino/blob/9526257c3d4c104c81e3ce31fc1ef485e75ae5e7/docs/manage-fileset-metadata-using-gravitino.md?plain=1#L247-L324


Maybe you should move this part to `manage-fileset-metadata-using-gravitino.md`, since it describes the usage of filesets.